src/features/user/MFV2/MyContributions/RegistrationItemCard.tsx

```diff
@@ -0,0 +1,41 @@
+import { ComponentProps } from 'react';
+import getImageUrl from '../../../../utils/getImageURL';
+import {
+  MyContributionsSingleRegistration,
+  MyForestProject,
+} from '../../../common/types/myForestv2';
+import ItemImage from './ItemImage';
+import styles from './MyContributions.module.scss';
+import RegistrationSummary from './RegistrationSummary';
+
+interface Props {
+  project?: MyForestProject;
+  contributionDetails: MyContributionsSingleRegistration;
+}
+
+export type RegistrationItemCardProps = ComponentProps<
+  typeof RegistrationItemCard
+>;
+
+const RegistrationItemCard = ({ project, contributionDetails }: Props) => {
+  const imageSource = project?.image
+    ? getImageUrl('project', 'medium', project.image)
+    : '';
+
+  return (
+    <article className={styles.registrationItemCard}>
+      <div className={styles.cardContent}>
+        <ItemImage {...(project ? { imageUrl: imageSource } : {})} />
+        <div className={styles.itemInfo}>
+          <RegistrationSummary
+            treeCount={contributionDetails.totalContributionUnits}
+            country={contributionDetails.country}
+            registrationDate={contributionDetails.contributions[0].plantDate}
+          />
```

Comment:
Not needed, as it does not impact readability since contributionDetails is used just thrice, and `contributionDetails.country` is far clearer than just `country` as I immediately know where the data is sourced from.